packages/french-verbs/test/conjugation.js

```diff
@@ -9,12 +9,12 @@ const FrenchVerbs = require('../dist/index.js');
 const Lefff = require('french-verbs-lefff/dist/conjugations.json');
 
 const testCasesConjugation = [
-  ['est allée', { verb: 'aller', person: 2, gender: 'F', aux: 'ETRE', tense: 'PASSE_COMPOSE', agreeGender: 'F' }],
-  ['finit', { verb: 'finir', person: 2, gender: 'M', tense: 'PRESENT' }],
-  ['est', { verb: 'être', person: 2, gender: 'M', tense: 'PRESENT' }],
-  ['sont', { verb: 'être', person: 5, gender: 'M', tense: 'PRESENT' }],
-  ['oignent', { verb: 'oindre', person: 5, gender: 'M', tense: 'PRESENT' }],
-  ['chantent', { verb: 'chanter', person: 5, gender: 'M', tense: 'PRESENT' }],
+  ['est allée', { verb: 'aller', person: 2, aux: 'ETRE', tense: 'PASSE_COMPOSE', agreeGender: 'F' }],

```

Comment:
change from 'gender' to 'agreeGender': I understand that it was a mistake in the original file, and that this is debug. Can you confirm?

Reply:
yes, i confirm